helm/metrics-server-app/templates/ciliumnetworkpolicy.yaml

```diff
@@ -14,4 +14,7 @@ spec:
     - toEntities:
         - kube-apiserver
         - cluster
+  ingress:
```

Comment:
Why is that needed ? 🤔

Reply:
a message to your team's channel with a detailed explanation is coming.

Reply:
You're scaring me :D

I think we never added ingress because no component should scrape that but I guess the api-server is?